systems/plants/@Manipulator/Manipulator.m

```diff
@@ -497,9 +497,13 @@ function num_contacts(obj)
       con = BoundingBoxConstraint(jl_min,jl_max);
       con = setName(con,cellfun(@(a) [a,'_limit'],obj.getPositionFrame.coordinates,'UniformOutput',false));
       if isempty(obj.joint_limit_constraint_id)
-        [obj,id] = addStateConstraint(obj,con,1:obj.getNumPositions);
-        obj.joint_limit_constraint_id = id;
+        if any(jl_min~=-inf) || any(jl_max~=inf)
```

Comment:
I agree it is more specific, but if someone sets the lower limit to +inf, we probably do not want to add that constraint either.